src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/integration/TestBlockOperators.java

```diff
@@ -153,36 +155,48 @@ public void testInvalidInputTypeModName() throws EvaluationException {
      * ----------------------------------- SOUNDS -----------------------------------
      */
 
-    @IntegrationTest
+    @Test
     public void testBlockSound() throws EvaluationException {
+        // net.minecraft.util.SoundEvent.getSoundName is physical client only
+        // TODO There is probably a better way to detect this, but this works!
```

Comment:
I suggest hardcoding the sound event names, so that this test can also run client-side.